apps/client/src/pages/ConclusionPage.js

```diff
@@ -112,16 +106,33 @@ const ConclusionPage = ({ topic, checker }) => {
             Datum
           </Heading>
           <Paragraph>{currentDateTime} uur.</Paragraph>
-
-          <Heading forwardedAs="h2">Adresgegevens</Heading>
-          <Paragraph gutterBottom={30}>
-            {streetName} {houseNumberFull}
-            <br />
-            {postalCode} {residence}
+          {address && (
+            <>
+              <Heading forwardedAs="h2">Adresgegevens</Heading>
+              <Paragraph gutterBottom={30}>
+                {address.streetName} {address.houseNumberFull}
+                <br />
+                {address.postalCode} {address.residence}
+              </Paragraph>
+              <Paragraph>
+                Deze informatie hebben we gebruikt bij het invullen van de
+                check:
+              </Paragraph>
+
+              <AddressData displayZoningPlans={false} address={address} />
+            </>
+          )}
+
+          <Heading forwardedAs="h2">Uw antwoorden</Heading>
+          <Paragraph>
+            Hieronder kunt u per vraag uw gegeven antwoord teruglezen.
           </Paragraph>
+          <QuestionAnswerTable checker={checker} />
```

Comment:
In the Print version we want the user to see the results in this order: Address, Conclusion, Questions/Answers. Now you swapped it around. Can you place that back?

Reply:
Not sure what I did here. Fixed.